src/vs/workbench/browser/parts/notifications/notificationsToasts.ts

```diff
@@ -421,15 +421,19 @@ export class NotificationsToasts extends Themable {
 
 	protected updateStyles(): void {
 		this.mapNotificationToToast.forEach(t => {
-			const backgroundColor = this.getColor(NOTIFICATIONS_BACKGROUND);
-			t.toast.style.background = backgroundColor ? backgroundColor : '';
-
 			const widgetShadowColor = this.getColor(widgetShadow);
 			t.toast.style.boxShadow = widgetShadowColor ? `0 0px 8px ${widgetShadowColor}` : '';
 
 			const borderColor = this.getColor(NOTIFICATIONS_TOAST_BORDER);
 			t.toast.style.border = borderColor ? `1px solid ${borderColor}` : '';
 		});
+
+		document.querySelectorAll('.notification-toast-container .monaco-list-rows')
+			.forEach((element: HTMLElement) => {
+				const backgroundColor = this.getColor(NOTIFICATIONS_TOAST_BACKGROUND);
+				element.style.background = backgroundColor ? backgroundColor : '';
```

Comment:
ternary operator can be replaced with logical "OR". resulted statement will be shorter

Reply:
thanks for the code review!
I wanted to keep the code style consistent, for example here,
https://github.com/microsoft/vscode/blob/fc1c307d093163d6355569a79bb6639e534d8934/src/vs/workbench/browser/parts/notifications/notificationsList.ts#L221-L232

Do you think shortening this statement will decrease readability at all?